### PR TITLE
Initial import of the WIP from https://github.com/prometheus-community/helm-charts/pull/3872

### PR DESCRIPTION
**Pull request by mattray**
The Prometheus OpenCost Exporter Helm chart needs a primary source.

### CODE REVIEW

**Comment by mattray** [comment]
Fixes https://github.com/opencost/opencost-helm-chart/issues/128

**Review by toscott — COMMENTED** [review]
Reading through the comments on the prometheus-community helm chart pull request, I'm a little confused on what we are trying to accomplish with this. I would expect the home url monotek was referencing to point to the source application that the community helm chart is installing, not another helm chart. I would expect to just set that to: 
https://github.com/opencost/opencost

**Comment by toscott** [comment]
I guess just other opinions - I don't think that this necessarily fits into the prometheus-XXX-exporter model. If OpenCost was a product that did not produce prometheus metrics natively and we built a second tool just for exporting metrics, it would make sense to have a prometheus-opencost-exporter helm chart. Since OpenCost exports metrics natively, I don't think we need a new helm chart?

File: charts/prometheus-opencost-exporter/tmp/prometheus-opencost-exporter/Chart.yaml
[the remark this answers is not in the document]
Comment (mattray) [reply charts/prometheus-opencost-exporter/tmp/prometheus-opencost-exporter/Chart.yaml:1]: Accidental commits, removed.

**Comment by mattray** [comment]
@toscott I'm don't have a good understanding what they want in the Prometheus Community hosted version of the chart. I'm happy to make that the only location for it and change it to https://github.com/opencost/opencost and close this out.

I think it belongs in the prometheus-XXX-exporter model because some folks only use OpenCost as a metric exporter and don't use the UI at all. https://www.opencost.io/docs/integrations/opencost-exporter

**Comment by toscott** [comment]
> @toscott I'm don't have a good understanding what they want in the Prometheus Community hosted version of the chart. I'm happy to make that the only location for it and change it to https://github.com/opencost/opencost and close this out.

Based on other charts in that repository, I think specifically what they were after on the other PR for the "home" field was a link to the underlying application code of the exporter: e.g.  https://github.com/opencost/opencost. 

> I think it belongs in the prometheus-XXX-exporter model because some folks only use OpenCost as a metric exporter and don't use the UI at all. https://www.opencost.io/docs/integrations/opencost-exporter

Obviously up to you. Personally I'd think we wouldn't want to try to maintain two separate helm charts, but rather a single helm chart and just make sure its straightforward to use it with a disabled UI component. (This is actually how we are using it at my company already). I suspect splitting off a new helm chart will also split up community involvement between the two and result in inconsistent feature support between the charts.

**Pull request closed without merging**